packages/salesforcedx-vscode-core/src/commands/forceOpenDocumentation.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import {
+  APEX_FILE_NAME_EXTENSION,
+  SOQL_FILE_NAME_EXTENSION
+} from '../constants';
+import { nls } from '../messages';
+
+const auraPath = '/force-app/main/default/aura/';
+const apexClassesPath = '/force-app/main/default/classes/';
+const lwcPath = '/force-app/main/default/lwc/';
+const functionsPath = '/functions/';
```

Comment:
nit: FWIW I would have also put these in the constants file.  Same deal where any time we need the auraPath we should always use the same value across the codebase.  First other place it could be used is in the unit tests to build the stub responses.

Reply:
I'll go and move them